ts/Stock/Indicators/Stochastic/StochasticIndicator.ts

```diff
@@ -206,7 +206,7 @@ class StochasticIndicator extends SMAIndicator {
             K = CL / HL * 100;
 
             xData.push(xVal[i]);
-            yData.push([K, null]);
+            yData.push([isNaN(K) ? yData[yData.length - 1][0] : K, null]);
```

Comment:
Can we add a few more conditions here?
1. When CL = 0, then we can render the line at 0 anyway
2. When it's a bug for the first point, then `yData[yData.length - 1]` will be undefined, so use simply `null`, but please check it (+test)

I couldn't find an official statement from the Stochastic's author in his paperwork about what should be done in this case, but repeating the value seems to be a reasonable solution (if the price is frozen, then the indicator will be too)